plugins/flutter_aepcore/test/flutter_aepcore_test.dart

```diff
@@ -182,19 +182,19 @@ void main() {
     });
 
     test('invokes correct method', () async {
-      await MobileCore.dispatchEventWithResponseCallback(expectedEvent);
+      await MobileCore.dispatchEventWithResponseCallback(expectedEvent, 1000);
 
       expect(log, <Matcher>[
         isMethodCall(
           'dispatchEventWithResponseCallback',
-          arguments: eventConstructorData,
+          arguments: {"eventData": eventConstructorData, "timeout": 1000},
         ),
       ]);
     });
 
     test('returns correct result', () async {
-      final actualEvent =
-          await MobileCore.dispatchEventWithResponseCallback(expectedEvent);
+      final actualEvent = await MobileCore.dispatchEventWithResponseCallback(
+          expectedEvent, 1000);
       expect(actualEvent.eventName, returnedEvent.eventName);
     });
   });
```

Comment:
Can you add a couple of more tests to verify the new error checks added [here](plugins/flutter_aepcore/android/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepcore/FlutterAEPCorePlugin.java)

Reply:
I checked the Dart test code and didn't find any error handling tests. Then, I noticed that the Dart tests were set up to mock the native calls, so we can't test the native code in the current unit tests. We may later cover those in the functional tests.